app/services/session_service.rb

```diff
@@ -4,7 +4,7 @@ class TokenNotRecognised < StandardError; end
   class InvalidTokenAndIdentifierCombination < StandardError; end
 
   def self.send_magic_link_email!(email_address)
-    if (user = find_user_by_email(email_address))
+    if (user = find_user_by_email(email_address.downcase))
```

Comment:
The `downcase` shouldn't be necessary, because `find_user_by_email` applies `downcase` too:

```ruby
  def self.find_user_by_email(email_address)
    User.where('lower(email_address) = ?', email_address.downcase).first
  end
```

Reply:
I've removed the `.downcase`, and changed the name of `find_user_by_email` to `find_user_by_lowercase_email` to be more explicit